### PR TITLE
Add feature to support line number args

### PR DESCRIPTION
**Pull request by shun095**
This PR adds feature to support following command
```
$EDITOR +10 file.txt
```
Just I would like to make it works better with tig: https://github.com/jonas/tig :)

### CODE REVIEW

File: plugin/tvim [hunk 1]
```diff
@@ -12,6 +12,11 @@ for file; do
     continue
   fi
 
+  if [[ "${file}" =~ ^\+[0-9]+$ ]]; then
+    opts="${opt:-} ${file}"
```
Comment (kyoh86) [inline plugin/tvim]:
```suggestion
    opts="${opts:-} ${file}"
```

**Review by kyoh86 — CHANGES_REQUESTED** [review]
nits

**Comment by shun095** [comment]
Sorry, thank you very much for your reviewing!

File: autoload/editerm.vim [hunk 2]
```diff
@@ -38,7 +38,7 @@ function! editerm#release_remote(bufnum, ret)
   unlet s:remotes[a:bufnum]
 endfunction
 
-function! editerm#open(command, lockfile, filename)
+function! editerm#open(command, lockfile, filename, options)
```
Comment (kyoh86) [inline autoload/editerm.vim:41]:
```suggestion
function! editerm#open(command, lockfile='', filename='', options='')
```

File: plugin/editerm.vim [hunk 3]
```diff
@@ -16,8 +16,8 @@ scriptversion 3
 " Description: It should be called from :terminal with three parameters
 " [(n|e|v|t), (lock file), (editing file)] as $EDITOR.
 function! Tapi_EditermEditFile(bufnum, arglist)
-  if len(a:arglist) == 3
-    call editerm#open(a:arglist[0], a:arglist[1], a:arglist[2])
+  if len(a:arglist) == 4
+    call editerm#open(a:arglist[0], a:arglist[1], a:arglist[2], a:arglist[3])
```
Comment (kyoh86) [inline plugin/editerm.vim:20]:
```suggestion
  if len(a:arglist) <= 4
    call call('editerm#open', a:arglist)
```

File: plugin/tdrop [hunk 4]
```diff
@@ -8,7 +8,7 @@ if [ $# -eq 0 ]; then
   if [ "$VIM_EDITERM_OPENER" = 'n' ] || \
      [ "$VIM_EDITERM_OPENER" = 'v' ] || \
      [ "$VIM_EDITERM_OPENER" = 't' ] ; then
-    echo -e "\033]51;[\"call\", \"Tapi_EditermEditFile\", [\"${VIM_EDITERM_OPENER}\", \"\", \"\"]]\007"
+    echo -e "\033]51;[\"call\", \"Tapi_EditermEditFile\", [\"${VIM_EDITERM_OPENER}\", \"\", \"\", \"\"]]\007"
```
Comment (kyoh86) [inline plugin/tdrop:11]:
```suggestion
    echo -e "\033]51;[\"call\", \"Tapi_EditermEditFile\", [\"${VIM_EDITERM_OPENER}\"]]\007"
```

File: plugin/tdrop [hunk 5]
```diff
@@ -17,5 +17,5 @@ for file; do
     file="${PWD%/}/${file}"
   fi
 
-  echo -e "\033]51;[\"call\", \"Tapi_EditermEditFile\", [\"${VIM_EDITERM_OPENER}\", \"\", \"$file\"]]\007"
+  echo -e "\033]51;[\"call\", \"Tapi_EditermEditFile\", [\"${VIM_EDITERM_OPENER}\", \"\", \"$file\", \"\"]]\007"
```
Comment (kyoh86) [inline plugin/tdrop:20]:
```suggestion
  echo -e "\033]51;[\"call\", \"Tapi_EditermEditFile\", [\"${VIM_EDITERM_OPENER}\", \"\", \"$file\"]]\007"
```

**Review by kyoh86 — APPROVED** [review]
LGTM with nits

**Pull request merged**